Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apt update && apt upgrade -y
 # Install requirements
 RUN apt update && apt install -y graphviz libffi6 libffi-dev libssl-dev cron
 RUN pip3 install --upgrade setuptools # otherwise cairocffi cannot be installed (https://github.com/lief-project/LIEF/issues/108)
+RUN pip3 install --upgrade datajoint # fixes reconnection bug
```

Comment:
The fix is present in the dev release of DataJoint, which needs an additional `--pre` flag to install. I strongly recommend you test it out first to ensure that this doesn't break anything as the dev releases do contain experimental features that are backward incompatible.